Makefile

```diff
@@ -7,10 +7,8 @@ ALL_DOCS := $(shell find . -name '*.md' -type f | sort)
 # All directories with go.mod files. Used in go mod tidy.
 ALL_GO_MOD_DIRS := $(shell find . -type f -name 'go.mod' -exec dirname {} \; | sort)
 
-GOTEST=go test
-GOTEST_OPT?=-v -timeout 30s
-GOTEST_OPT_WITH_RACE     = $(GOTEST_OPT) -race
-GOTEST_OPT_WITH_COVERAGE = $(GOTEST_OPT) -coverprofile=coverage.txt -covermode=atomic
+GOTEST=go test -v -timeout 30s -race
+GOTEST_WITH_COVERAGE = $(GOTEST) -coverprofile=coverage.txt -covermode=atomic
```

Comment:
The usages of  `GOTEST_OPT_WITH_COVERAGE ` should be updated to `GOTEST_WITH_COVERAGE` that is probably the reason why it is failing on the CI.

Reply:
Not sure. Why is there a timeout for coverage testing?

Reply:
Thanks @paivagustavo!

@jmacd there always has been. Whether we want moving forward is a separate question 😂